include/alpaka/core/Common.hpp

```diff
@@ -97,7 +111,21 @@
 //! In contrast to ordinary variables, you can not define such variables
 //! as static compilation unit local variables with internal linkage
 //! because this is forbidden by CUDA.
-#if(BOOST_LANG_CUDA && BOOST_ARCH_PTX) || (BOOST_LANG_HIP && (BOOST_ARCH_HSA || BOOST_ARCH_PTX))
+//!
+//! \attention It is not allowed to initialize the variable together with the declaration.
+//!            To initialize the variable alpaka::createStaticDevMemView and alpaka::memcpy must be used.
```

Comment:
```suggestion
//!            To initialize the variable alpaka::createStaticDevMemView an alpaka::memcpy must be used.
```

Reply:
`and` is fine at this point. You must create the view and use `alpaka::memcpy` to initialize the instance.

Reply:
Ok then I misread the sentence.